packages/salesforcedx-vscode-lwc/src/commands/createResources.ts

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
This file should be moved to a different module since it doesn't fit under `commands`, which hosts code that exposes command palette functionality. You can create a new one called `metaSupport` or something that reflects the purpose of the code's purpose.

Once you move it there you should surface the methods using an `index.ts` file. Take a look at how it's being done in the `telemetry` module or on other modules under `salesforcedx-vscode-core`